src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotificationDefendantOfHearingHandlerTest.java

```diff
@@ -170,6 +170,6 @@ private Map<String, String> getNotificationDataMapDef2(CaseData caseData) {
     void shouldReturnCorrectCamundaActivityId_whenInvoked() {
         assertThat(handler.camundaActivityId(CallbackParamsBuilder.builder().request(CallbackRequest
                                                                                          .builder().eventId(
-                "NOTIFY_DEFENDANT_HEARING").build()).build())).isEqualTo(TASK_ID_DEFENDANT);
+                "NOTIFY_DEFENDANT_HEARING").build()).build())).isEqualTo(TASK_ID_DEFENDANT1);
```

Comment:
This unit test fails.

NOTIFY_DEFENDANT_HEARING should be NOTIFY_DEFENDANT1_HEARING.